packages/lumx-react/src/components/flag/Flag.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Theme, ColorPalette, Icon } from '@lumx/react';
+import { GenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/utils';
+import { ColorVariant } from '..';
+
+export interface FlagProps extends GenericProps {
+    /** Theme adapting the component to light or dark background. */
+    theme?: Theme;
+    /** The color of the component */
+    color?: ColorPalette;
+    /** icon */
+    icon?: string;
+    /** label of the flag */
+    label: string;
+}
+
+const COMPONENT_NAME = 'Flag';
+export const CLASSNAME = getRootClassName(COMPONENT_NAME);
+const DEFAULT_PROPS: Partial<FlagProps> = {
+    theme: Theme.light,
+};
+
+/**
+ * Flag component.
+ *
+ * @param  props Component props.
+ * @return React element.
+ */
+export const Flag: React.FC<FlagProps> = ({ label, icon, color, className, theme }) => {
```

Comment:
Could you add a `forwardRef`? Every component should be able to forward a ref to be usable in some context which requires a ref like using this component as an anchor for a popover or a dropzone for a drag and drop feature, and so on...

You can look at the Badge component for example, except that in your case the type of your ref would be `HTMLSpanElement`.